cmake/ArangoDBInstall.cmake

```diff
@@ -81,19 +81,14 @@ install(
     ${PROJECT_SOURCE_DIR}/js/apps
     ${PROJECT_SOURCE_DIR}/js/server
   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR_ARANGO}/${ARANGODB_JS_VERSION}
-  REGEX       "^.*/aardvark/APP/.*/react/.*$"              EXCLUDE
+  REGEX       "^.*/aardvark/APP/.*/.*$"                    EXCLUDE
```

Comment:
does that keep files in "APP" direcotry? 
e.g. maniefest which is in `aardvark/APP/manifest.json`